packages/backend-common/src/scm/git.ts

```diff
@@ -111,8 +111,7 @@ export class Git {
     this.config.logger?.info(
       `Committing file to repo {dir=${dir},message=${message}}`,
     );
-
-    return git.commit({ fs, dir, message, author, committer });
+    return await git.commit({ fs, dir, message, author, committer });
```

Comment:
There's no effective change here as the `await` in this location doesn't add anything extra. Perhaps drop this but still bump `isomorphic-git` and change the changeset to reflect that?